### DISCUSSION TITLE
Excavate Command

### DISCUSSION THREAD

**Discussion by SteveMan67** [cc-tweaked/CC-Tweaked#1976] [Q&A]
I am using CC Tweaked with an ore harvester mod, and whenever the excavate program runs into an ore it takes a while to break so the program stops.

**Comment by fatboychummy** [comment]
Can you explain in more detail what is going on here? What ore harvesting mod? What is the error? What ore is it getting stuck on?

**Comment by SteveMan67** [comment]
I'm using ore harvester by Serilium. There is a delay whenever you try to break an ore block, and the turtle thinks it has hit bedrock because it takes a while to mine.

**Reply by fatboychummy** [reply]
> and the turtle thinks it has hit bedrock
As in the turtle prints "Returning to surface..."?

Can you place a turtle in front of an ore block, run the program `lua`, then run `turtle.dig()`? It should give the reason it is failing to dig, but the excavate program does not display the reason (it just detects it failed and returns to the surface).

```
CraftOS 1.9
Some message of the day!
> lua
Interactive Lua prompt.
Call exit() to exit.
lua> turtle.dig()
false
"Some error message"
```
It should look something like the above.

**Reply by fatboychummy** [reply]
Also, looking at https://modrinth.com/mod/ore-harvester, I would recommend making sure `oreHarvestWithoutSneak` in the config is set to `false`. Turtles cannot sneak, so they shouldn't trigger the ore harvesting mod if you have that set to false.

**Reply by SteveMan67** [reply]
That is set to false. I'll test it out and see what it does.
